hypertrace-gradle-publish-maven-central-plugin/README.md

```diff
@@ -15,13 +15,19 @@ For signing artifacts, following properties must be provided as gradle propertie
 - signingKey
 - signingPassword
 
+The plugin also uses [gradle-nexus-staging-plugin](https://github.com/Codearte/gradle-nexus-staging-plugin) plugin for closing and releasing staged repositories.
+
 Each property described below can be configured in the DSL. The default values are shown for each property,
 all of which, with the exception of license, can be omitted if left unchanged.
 ```kotlin
 hypertracePublishMavenCentral {
   license // REQUIRED to be a value defined in org.hypertrace.gradle.publishing.License
   repoName // REQUIRED. Name of the repository.
   url // Optional. The URL for the publication represented by the POM.
+  packageGroup // Optional. The package group as registered in Nexus staging profile.
+  stagingProfileId // Optional. The staging profile used to release given project.
```

Comment:
Do we have a requirement to change any of these from project to project? If not, I wouldn't expose them - we want to make it as fool-proof as possible, especially for publishing. Package group in particular, I would think, would come from the project (referenced already on line 163 - assuming those need to match).

Reply:
All of these properties are optional and does not need to be configured. I've exposed them only for debugging purpose.

`packageGroup` need to match with the package group registered with sonatype repository i.e `org.hypertrace` which is set as default value. I am exploring ways to have it match with project group i.e. `org.hypertrace.agent`.

Reply:
Given that the package group should never be changed then, wouldn't it make sense to not even expose it? Being able to debug is good, but making it simple and hard to misconfigure is better.